libs/constrained-generators/test/Constrained/Test.hs

```diff
@@ -269,25 +266,26 @@ testSpecNoShrink :: HasSpec fn a => String -> Specification fn a -> Spec
 testSpecNoShrink = testSpec' False
 
 testSpec' :: HasSpec fn a => Bool -> String -> Specification fn a -> Spec
-testSpec' withShrink n s =
+testSpec' withShrink n s = do
+  let checkCoverage' = checkCoverageWith stdConfidence {certainty = 1e6}
```

Comment:
This syntax is much clearer for integral types and waaaaay more common. I didn't even know about `NumDecimals` language extension.
```suggestion
  let checkCoverage' = checkCoverageWith stdConfidence {certainty = 1_000_000}
```

Reply:
I agree, this is better.

I had it that way because the default QuickCheck value is `1e9`, but most people won't know that so the `NumericUnderscores` approach is better.

I wish `NumericUnderscores` was on by default in ghc.